### PR TITLE
Bugfix: Fix invisible Kodi server settings

### PR DESCRIPTION
**Pull request by wutschel**
## Description

For the Kodi settings table there is a unique animation on iPhone. The animation slides in the table from bottom to top. For this we need to have a helper method which animates x and y origins.

## Summary for release notes

Bugfix: Fix invisible Kodi server settings

### CODE REVIEW

File: XBMC Remote/Utilities.m [hunk 1]
```diff
@@ -1021,6 +1021,18 @@ + (void)AnimView:(UIView*)view AnimDuration:(NSTimeInterval)seconds Alpha:(CGFlo
                      completion:^(BOOL finished) {}];
 }
 
++ (void)AnimView:(UIView*)view AnimDuration:(NSTimeInterval)seconds Alpha:(CGFloat)alphavalue XPos:(int)X YPos:(int)Y {
+    [UIView beginAnimations:nil context:nil];
```
Comment (kambala-decapitator) [inline XBMC Remote/Utilities.m:1025]: why old style?

File: XBMC Remote/Utilities.m [hunk 2]
```diff
@@ -1021,6 +1021,18 @@ + (void)AnimView:(UIView*)view AnimDuration:(NSTimeInterval)seconds Alpha:(CGFlo
...
+    [UIView setAnimationDuration:seconds];
+    view.alpha = alphavalue;
+    CGRect frame;
+    frame = view.frame;
```
Comment (kambala-decapitator) [inline XBMC Remote/Utilities.m:1029]:
```suggestion
    CGRect frame = view.frame;
```

**Review by kambala-decapitator — CHANGES_REQUESTED** [review]

File: XBMC Remote/Utilities.m (hunk 1 above)
Comment (wutschel) [reply XBMC Remote/Utilities.m:1025]: Oh, that was because I searched for the error in the old code after bi-secting. Will change this.

**Comment by wutschel** [comment]
Done and already squashed.

**Pull request merged**